src/components/Modal/Example.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div>
+    <button @click="openModal()">coucou</button>
+    <modal :is-open="isOpen" :show-close-button="true">
+    </modal>
```

Comment:
```suggestion
    <modal :is-open="isOpen" :show-close-button="true" />
```

Les balises se simplifient quand tu n'as pas de contenu. Même si je pense que tu devrais en mettre